src/assessment.js

```diff
@@ -27,7 +27,7 @@ var q1 = /* your answer here */;
 //
 //     `['alderaan', 'deathStar', 'fire', 'global']`
 //
-var q2 = [/* your answer here */];
+var q2 = ['deathStar', 'fire', 'global'];
```

Comment:
Not quite. `laserQuantity` is defined inside the `deathStar` function. Will it be reachable if someone tries to access it from outside that function?